luma/core/device.py

```diff
@@ -90,6 +91,61 @@ def cleanup(self):
         self._serial_interface.cleanup()
 
 
+class parallel_device(device):
+    """
+    Wrapper class to manage communications with devices that can operate in
+    four or eight bit modes.
+
+    .. versionadded:: 1.15.1
```

Comment:
I think it'll be 1.16.0.

Reply:
Changed all of the 1.15.1 instances to 1.16.0